config/config.go

```diff
@@ -358,6 +371,50 @@ func parseDatabaseURL(cfg *Prest) {
 	}
 }
 
+// fetchJWKS tries to get the JWKS from the URL in the config
+func fetchJWKS(cfg *Prest) {
+	if cfg.JWTWellKnown == "" {
+		log.Debugln("no JWT WellKnown url found, skipping")
+		return
+	}
+	if cfg.JWTJWKS != "" {
+		log.Debugln("JWKS already set, skipping")
+		return
+	}
+
+	// Call provider to obtain .well-known config
+	r, err := http.Get(cfg.JWTWellKnown)
```

Comment:
please add context to this call with a 5s timeout

Reply:
Done in latest commit.
https://github.com/prest/prest/pull/871/commits/fee30e7ae1162e49e1d88f7fa384c73b016fdc95